ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -546,6 +549,36 @@ route_maps:
           - "< set rule 2 as string >"
 ```
 
+### Groups
+
+#### Group Interfaces
+
+```yaml
+group:
+  interface:
+    < group_name >:
+      interfaces:
+        - "< interface_or_interface_range >"
+      maintenance_profile:
+        bgp: < bgp_maintenance_profile_name >
+        interface: < interface_maintenance_profile_name >
+```
+
+#### BGP Interfaces
+
+```yaml
+group:
+  bgp:
+    < group_name >:
+      vrf: "< vrf_name >"
+      neighbor:
+        - "< ip_address >"
+        - "< ipv6_address >"
+        - "< peer_group_name >"
+      maintenance_profile:
+        bgp: < bgp_maintenance_profile_name >
```

Comment:
It is possible to configure more than one maintenance profile bgp per group (same for interface):
```eos
group interface test
   interface Et10-20,30-40
   maintenance profile bgp test2
   maintenance profile bgp test
   exit
```
So we should make these into lists.